assemblies/installing-odh-v1.adoc

```diff
@@ -0,0 +1,30 @@
+:_module-type: ASSEMBLY
+//pv2hash: 
+
+ifdef::context[:parent-context: {context}]
+
+:context: installv1
+
+[id="installing-odh-v1_{context}"]
+= Installing Open Data Hub (v1)
+
+[role='_abstract']
+You can install the Open Data Hub Operator version 1 on your OpenShift Container Platform from the OpenShift web console. For information about installing the Open Hub Operator version 2, see link:{odhdocshome}/installing-odh/#installing-odh-v2[Installing Open Data Hub (v2)].
```

Comment:
Given that the bullet list below introduces a number of installation tasks (of which installing the Operator is just one), can we remove the focus on the Operator from the abstract? That is:

`You can install Open Data Hub Operator version 1 on your OpenShift Container Platform cluster by using the OpenShift web console. For information about installing the Open Hub version 2, see link:{odhdocshome}/installing-odh/#installing-odh-v2[Installing Open Data Hub (v2)].`